controldev_websocket.orogen

```diff
@@ -1,15 +1,23 @@
-name "controldev_websocket"
+name 'controldev_websocket'
 
 using_library 'controldev'
+using_library 'base-logging'
 using_library 'jsoncpp', typekit: false
 
-import_types_from "std"
-import_types_from "base"
+import_types_from 'base'
+import_types_from 'controldev/RawCommand.hpp'
+import_types_from 'Mapping.hpp'
 
-task_context "Task" do
+task_context 'Task' do
     property 'port', 'uint16_t', 58000
+    property 'axis_map', '/std/vector</controldev_websocket/Mapping'
```

Comment:
It is. At controldev.orogen is also the same way and I have to check again if the correct way works because I remeber I chaged it (maybe the change wasn't effective).

Reply:
It worked in the correct way